ext/opentelemetry-ext-django/setup.cfg

```diff
@@ -38,7 +38,7 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    django >= 2.2
+    django >= 1.10
```

Comment:
Looking at the pypi release history for django 1.10, it looks like it requires python 3.4 or 3.5, should the `python_requires` and `classifiers` be updated to support those versions of Python as well?

Reply:
Good point!
Tests are passed already and i don't see any restrictions to do this correction